test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -1,184 +1,109 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Basic Send-MailMessage tests" -Tags CI {
+
+Describe "Send-MailMessage" -Tags CI, RequireSudoOnUnix {
     BeforeAll {
-        function test-smtpserver
-        {
-            $rv = $false
+        Register-PackageSource -Name nuget.org -Location https://www.nuget.org/api/v2 -ProviderName NuGet -ErrorAction SilentlyContinue
 
-            try
-            {
-                $tc = New-Object -TypeName System.Net.Sockets.TcpClient -ArgumentList "localhost", 25
-                $rv = $tc.Connected
-                $tc.Close()
-            }
-            catch
-            {
-                $rv = false
-            }
+        $nugetPackage = "netDumbster"
+        Find-Package $nugetPackage -ProviderName NuGet | Install-Package -Scope CurrentUser -Force
```

Comment:
There can be small perf improvement here: 

```suggestion
       Install-Package -Name $nugetPackage -scope CurrentUser -Force
```

Reply:
@adityapatwardhan Thank you for the great suggestion. I've used your suggestion, but also added the ProviderName argument for Install-Package to explicitly use NuGet. I hope that is fine.
I've also replaced the API version for NuGet from V2 to V3 for Register-PackageSource, in order to use the latest and recommended API URL.